mobile/lib/modules/home/views/home_page.dart

```diff
@@ -220,8 +220,7 @@ class HomePage extends HookConsumerWidget {
         top: true,
         child: Stack(
           children: [
-            ref.watch(assetProvider).renderList == null ||
-                    ref.watch(assetProvider).allAssets.isEmpty
+            ref.watch(assetProvider).renderList == null
```

Comment:
With this change I somehow cannot see the `buildLoadingIndicator()` trigger

Reply:
Reverted to the old check.